deep_qa/contrib/layers/tree_composition_lstm.py

```diff
@@ -160,141 +160,146 @@ def build(self, input_shape):
         # of treeLSTM
         if self.implementation == 1:
             # Input dimensionality for all W2s is output_dim, and there are 5 W2s: i, fp, fa, u, o
-            self.W2 = self.add_weight((self.output_dim, 5 * self.output_dim),
+            self.W2 = self.add_weight(shape=(self.output_dim, 5 * self.output_dim),
                                       name='{}_W2'.format(self.name), initializer=self.initializer)
             # Input dimensionality for all U2s is output_dim, and there are 5 U2s: i, fp, fa, u, o
-            self.U2 = self.add_weight((self.output_dim, 5 * self.output_dim),
+            self.U2 = self.add_weight(shape=(self.output_dim, 5 * self.output_dim),
                                       name='{}_U2'.format(self.name), initializer=self.initializer)
 
             # Input dimensionality for all W3s is output_dim, and there are 6 W2s: i, fp, fa1, fa2, u, o
-            self.W3 = self.add_weight((self.output_dim, 6 * self.output_dim),
+            self.W3 = self.add_weight(shape=(self.output_dim, 6 * self.output_dim),
                                       name='{}_W3'.format(self.name), initializer=self.initializer)
             # Input dimensionality for all U3s is output_dim, and there are 6 U3s: i, fp, fa1, fa2, u, o
-            self.U3 = self.add_weight((self.output_dim, 6 * self.output_dim),
+            self.U3 = self.add_weight(shape=(self.output_dim, 6 * self.output_dim),
                                       name='{}_U3'.format(self.name), initializer=self.initializer)
             # Input dimensionality for all V3s is output_dim, and there are 6 V3s: i, fp, fa1, fa2, u, o
-            self.V3 = self.add_weight((self.output_dim, 6 * self.output_dim),
+            self.V3 = self.add_weight(shape=(self.output_dim, 6 * self.output_dim),
                                       name='{}_V3'.format(self.name), initializer=self.initializer)
 
             self.b2 = K.variable(np.hstack((np.zeros(self.output_dim),
-                                            K.get_value(self.add_weight(self.output_dim,
+                                            K.get_value(self.add_weight(shape=self.output_dim,
+                                                                        name='FIXME',
```

Comment:
I was just trying to fix the pylint error, and didn't want to think about this code.  I since thought about it a bit, and gave a kind-of-reasonable name.